plugins/package-managers/cocoapods/src/main/kotlin/Lockfile.kt

```diff
@@ -56,7 +56,7 @@ internal data class Lockfile(
 
 internal fun String.parseLockfile(): Lockfile {
     val root = Yaml.default.parseToYamlNode(this).yamlMap
-    val pods = root.get<YamlList>("PODS")!!.items.map { it.toPod() }
+    val pods = root.get<YamlList>("PODS")?.items.orEmpty().map { it.toPod() }
```

Comment:
This relates a bit to my comment on the first commit. While I'm fine with the change per se, I'd appreciate if this semantic change could already be done as a preparation on the existing code, before the migration, and then have no more semantic changes as part of the migration.

Reply:
I tried, but moving it before the migration looks a bit ugly to me. 
Also, to me the added value is not obvious. 
I can offer to either keep it here or drop it.